makefile

```diff
@@ -0,0 +1,59 @@
+# Download and install all dependencies packages
+#
+
+device-id-in-forwardlist := $(shell adb forward --list | wc -l)
```

Comment:
yes!...I check the device id in forward list when I start to write the makefile, but soon I found that we might not have such scenario for mtbf_driver....do we ?